README.md

```
@@ -92,34 +74,51 @@ Join our Discord community for updates, support, and games: https://discord.gg/D
+> **Note:** Django 6.0 requires Python 3.12 or higher. If you have multiple versions on Windows, use a compatible installed version, for example: `py -3.12 -m venv venv`.
+
+**On Windows:**
+```powershell
 python -m venv venv
-venv\Scripts\activate        # Windows
-source venv/bin/activate     # macOS / Linux
+venv\Scripts\Activate.ps1
+```
+
+If you see the error *"Script execution is disabled on this system"*, run the following command in PowerShell before activating:
+```powershell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)